rollup.config.js

```diff
@@ -1,20 +1,34 @@
 import babel from 'rollup-plugin-babel';
 import minify from 'rollup-plugin-babel-minify';
+import replace from 'rollup-plugin-replace';
+import nodeResolve from 'rollup-plugin-node-resolve';
 
 export default {
   input: 'src/index.js',
   plugins: [
+    nodeResolve(),
     babel({
       babelrc: false,
       presets: [['env', { modules: false }], 'stage-2', 'react'],
       plugins: [
         'external-helpers',
-        ['transform-react-remove-prop-types', { mode: 'wrap' }],
       ],
+      env: {
+        production: {
+          plugins: [
+            ['transform-react-remove-prop-types', { removeImport: true }],
+          ],
+        },
+      },
     }),
-    process.env.MINIFY ? minify() : false,
+    replace({
+      'process.env.NODE_ENV': JSON.stringify('production')
+    }),
+    process.env.NODE_ENV === 'production' ? minify({
+      comments: false,
+    }) : false,
   ].filter(Boolean),
-  external: ['react', 'react-dom', 'prop-types', 'popper.js'],
+  external: ['react', 'react-dom', 'prop-types'],
```

Comment:
We specifically want to leave popper.js out the build so that people can use their own version, mock it and reduce the final bundle size.

Why do you think it should be bundled?

Reply:
Because that's how Reactstrap and various other 3rd party libraries are using `react-popper` in a static sense. Essentially, think about a lot of frameworks and applications that are using React ... they want to add drop-in tooltip support and not worry about having to include an underlying tooltip api separately. 

As a compromise, I wouldn't mind creating 2 versions. That's actually what we do in Reactstrap as well ... we have a 'full.js' version.  That way people could include statically without popper.js implementation or a full drop-in that just works. Let me know if you're okay with that and I'll update as well as update the readme.